### PR TITLE
update readme: add some little changes on code style to JS

### PR DESCRIPTION
**Pull request by Nicholas-Kipkoech**

### CODE REVIEW

**Comment by Nicholas-Kipkoech** [comment]
@Nicanor008 check

File: README.md [hunk 1]
```diff
@@ -20,7 +20,7 @@ npm install nicanor-autocomplete
 ### Usage
 Here’s how to use the Autocomplete class with a simple suggestion provider:
 
-```
+``` Js
```
Comment (Nicanor008) [inline README.md:23]: nice

File: README.md [hunk 2]
```diff
@@ -111,7 +111,7 @@ new Autocomplete(inputElement, suggestionProvider, {
 #### HTML Structure
 Ensure you have an input element in your HTML:
 
-```
+``` Js
```
Comment (Nicanor008) [inline README.md:114]: this should be `html`

**Review by Nicanor008 — CHANGES_REQUESTED** [review]
thanks for this contribution @Nicholas-Kipkoech 

just a quick feedback here

if you don't mind also you can update the code on installation also 

Screenshot 2024-07-31 at 12 21 29

**Pull request closed without merging**